### PR TITLE
Created seed file (and added a line to the gemfile)

### PR DESCRIPTION
**Pull request by mattysplash**
Hi team,

Seed file added – it gives us 20 entries to test with.
Only other change was to the gemfile.rb which now has 'faker' added.

### CODE REVIEW

**Comment by tarakoomen** [comment]
Nice work there Matt, loving the creativity on the seed file and the pretty advanced use of Faker! 🙌 Let's change line 90 of seed.rb so that it includes your name too: puts "Hi Matt, Yumi & Tara!!" 😎

**Review by tarakoomen — COMMENTED** [review]
One additional question out of interest: did the Gemfile.lock automatically change after you added faker to the Gemfile? No biggie, just learning here

**Comment by mattysplash** [comment]
> One additional question out of interest: did the Gemfile.lock automatically change after you added faker to the Gemfile? No biggie, just learning here

Yep, I only went in and added faker to the gemfile, it seems that Gemfile.lock must update on its own

File: db/seeds.rb [hunk 1]
```diff
@@ -1,9 +1,98 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
-#     MovieGenre.find_or_create_by!(name: genre_name)
-#   end
+# Do we want to delete existing data each time
```
Comment (tarakoomen) [inline db/seeds.rb]: Good question! Seeing as we will also work with user inputs I'd leave this commented out for now. We can always delete manually in rails console if need be.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,9 +1,98 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
```
Comment (tarakoomen) [inline db/seeds.rb:7]: If you're updating this file this can go too ;) but not important

**Review by tarakoomen — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 2 above)
Comment (mattysplash) [reply db/seeds.rb:7]: cool, cool. I already deleted those lines. I think you see them here highlighted in red just as an fyi that they are now gone/changed.

The first line of my branched seed file starts at line 1 that you see here in green (..where I ask if we want to delete or not)

Comment (tarakoomen) [reply db/seeds.rb:7]: Awesome, that makes a lot of sense, thanks!

**Pull request merged**